doc/api/fs.md

```diff
@@ -685,7 +687,9 @@ changes:
 * `path` {string|Buffer|URL}
 * `mode` {integer}
 
-Synchronous chmod(2). Returns `undefined`.
+The synchronous version of [`fs.chmod()`][]. Returns `undefined`.
```

Comment:
I know it duplicates content, but it would be ideal if this wasn't just a link to something else... e.g. this could be

```markdown
Synchronously change the permissions of a file or throw if unable to do so.
```